<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertesWidget</class>
 <widget class="QWidget" name="AlertesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alertes paiement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1" rowspan="3">
      <widget class="QCalendarWidget" name="calendar"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="tableLabel">
       <property name="text">
        <string>Tableau des alertes, ID, et dates</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="alertesLabel">
       <property name="text">
        <string>Inscrire la nouvelle alerte</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" rowspan="2">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="recordButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>En&amp;registre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>E&amp;fface</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="alerteEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string>Utilisateurs</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="usersBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
